### DISCUSSION TITLE
How to select enclosing mark?

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#1835] [Q&A] [answered]
For instance:
```
func(() => {
 // contents
});  // select outter {}

const string = `some content`; // select everything within "``"
```

and so on... Like in vim, with: `vi{mark}` (or `:'<, '>`), given that the cursor is within the boundaries.

**Comment by the-mikedavis** [comment, marked as answer]
I think you're looking for `ma` (select around) and `mi` (select inside). `ma{`/`ma}` selects around the curly brackets in the first example and <code>mi`</code> selects inside the backtics in the second example.

**Reply by ghost** [reply]
Thanks! I should've gone to "[Surround](https://docs.helix-editor.com/usage.html#surround)" section first. 😅
